policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/ns_basic_personal_amount/basic_amount.yaml

```diff
@@ -0,0 +1,11 @@
+description: Nova Scotia basic amount in basic personal amount.
```

Comment:
Change this description to be:
Nova Scotia provides this basic amount under basic personal amount.